packages/syntax-core/src/SelectList/SelectList.test.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
@nathan-ong could we inline these disables per line?

That way we
* can count them 
* avoid accidentally disabling one

Reply:
ah, ok that's good to know the reasoning behind that